Makefile

```diff
@@ -130,14 +137,17 @@ util/format_converter_mac: src/main/c/binarize.cpp
 endif
 
 .PHONY: build-mindbender
+MINDBENDER=mindbender
 build-mindbender:
-	git submodule update --init mindbender
-	$(MAKE) -C mindbender clean-packages
-	$(MAKE) -C mindbender package
+	@util/build/build-submodule-if-needed $(MINDBENDER) mindbender-LATEST-Darwin-x86_64.sh
```

Comment:
mindbender build has been optional in the past as it's quite a big one, so I kept it as it is.  (stage.sh also optionally stages the built binary.)  I feel we should do that in a separate PR